CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# 0.2.0
+
+### New Features
+- [EKS] Add support for selecting NVIDIA Driver Version [PR #2](https://github.com/NVIDIA/nvidia-terraform-modules/pull/2)
+- [GKE] Add support for selecting NVIDIA Driver Version [PR #2](https://github.com/NVIDIA/nvidia-terraform-modules/pull/2)
+- [EKS]Increment version of GPU operator [PR #2](https://github.com/NVIDIA/nvidia-terraform-modules/pull/2)
+- [GKE]Increment version of GPU operator [PR #2](https://github.com/NVIDIA/nvidia-terraform-modules/pull/2)
+- [AKS] Add support for deploying cluster in existing resource group [PR #3](https://github.com/NVIDIA/nvidia-terraform-modules/pull/3)
+
+### Bug Fixes
+- [EKS] Remove kubeconfig script [PR #4](https://github.com/NVIDIA/nvidia-terraform-modules/pull/4)
+
+### Breaking Changes
+n/a
+
+### Other Changes
+n/a
+
+### Known Issues
+- [EKS] Default VPC subnet values do not enable cluster to scale to 100 nodes with default configuration. Considering updating in next release.
```

Comment:
Consider re-wording and I don't think 100 nodes or 50 or 73 nodes is relevant.  
"Default VPC subnet values do not enable cluster scaling after launch.  Config must indicate maximum nodes initially."